data-enricher/data-enricher.yaml

```
@@ -0,0 +1,232 @@
+              schema:
+                $ref: "#/components/schemas/Error"
+
+components:
+  securitySchemes:
+    BearerAuth:
+      description: This token is generated when a user enters credentials. Although you can leverage this token when making API calls, please note that due to security concerns it frequently expires.
+      type: http
+      scheme: bearer
+
+  schemas:
+    Error:
+      required:
+        - id
+        - status
+        - path
+        - message
+        - details
+      type: object
+      description: An error returned by Inperium API.
+      properties:
+        id:
+          type: string
+          format: uuid
+          description: A unique error ID.
+        status:
+          type: integer
+          description: The HTTP response code.
+          example: 403
+        path:
+          type: string
+          description: An URI that identifies the specific occurrence of the error.
+          example: "GET items/$OBJECT-UUID"
+        message:
+          type: string
+          description: The human-readable localized error message.
+          example: "Could not change an item."
+        details:
+          type: string
+          description: The human-readable explanation of the error for developers on English.
+          example: "Item entity $OBJECT-UUID has READ_ONLY status."
+
+    Person:
+      type: object
+      description: The information about a person.
+      properties:
+        fullName:
+          type: string
+          description: The full name of the individual.
+        ageRange:
+          type: string
+          description: An age range of the individual.
+        gender:
+          type: string
+          description: The gender of the individual.
+        title:
+          type: string
+          description: Job or position title.
+        organization:
+          type: string
+          description: The name of the organization/place of employment.
+        location:
+          type: string
+          description: The current location of the individual.
+        bio:
+          type: string
+          description: The bio of the individual.
+        emails:
+          type: array
+          description: Array representing additional known email addresses.
+          items:
+            $ref: "#/components/schemas/Email"
+        phones:
+          type: array
+          description: Array representing additional known phone numbers.
+          items:
+            $ref: "#/components/schemas/Phone"
+        name:
+          $ref: "#/components/schemas/Name"
+        locations:
+          type: array
+          description: Locations of the individual.
+          items:
+            $ref: "#/components/schemas/Location"
+        employment:
+          $ref: "#/components/schemas/Employment"
+        profiles:
+          type: array
+          description: Profiles on the social platforms.
+          items:
+            $ref: "#/components/schemas/Profile"
+        avatar:
```

```suggestion
        avatarUrl:
```
For consistency with Hub